### ISSUE TITLE
What is Application Accessibility?

### ISSUE THREAD

**Issue by beauremus** [fermi-ad/controls-issues#38]
### Describe the topic

Most people agree that everyone should be able to use our applications, but they may not know what goes into that or what to consider. I think an introduction to accessibility standards and common pitfalls would be helpful.

### Recommended presenter

Madelyn Polzin

### Topic Track

User Applications

### DocDB Link
https://beamdocs.fnal.gov/cgi-bin/sso/ShowDocument?docid=10240
